src/components/ciConfig/CIConfig.utils.ts

```diff
@@ -426,6 +450,14 @@ export const getCIConfigDiffValues = (
             if (gitMaterial.gitMaterialId === ciConfigOverride?.ciBuildConfig?.gitMaterialId) {
                 currentMaterialName = gitMaterial.materialName
             }
+
+            if (gitMaterial.gitMaterialId === globalCIConfig.ciBuildConfig?.buildContextGitMaterialId) {
```

Comment:
instead of multiple if we can use if else condition as only only condition will satisfy at a time

Reply:
no, all the conditions can be satisfied at a time.